lib/api/tests/js_instance.rs

```diff
@@ -535,12 +535,12 @@ mod js {
         let instance = Instance::new(&mut store, &module, &import_object).unwrap();
 
         let memory = instance.exports.get_memory("memory").unwrap();
-        assert_eq!(memory.data_size(&store), 65536);
-        let memory_val = memory.uint8view(&store).get_index(0);
+        assert_eq!(memory.view(&store).data_size(), 65536);
+        let memory_val = memory.view(&store).uint8view().get_index(0);
         assert_eq!(memory_val, 0);
 
-        memory.uint8view(&store).set_index(0, 2);
-        let memory_val = memory.uint8view(&store).get_index(0);
+        memory.view(&store).uint8view().set_index(0, 2);
```

Comment:
Let's try to use the MemoryView API directly, so we can remove the `uint8view` dependency

Reply:
done